core-web/libs/block-editor/src/lib/extensions/ai-image-prompt/components/ai-image-prompt-form/ai-image-prompt-form.component.ts

```diff
@@ -125,6 +128,10 @@ export class AiImagePromptFormComponent implements OnChanges, OnInit {
 
     copyToClipboard(): void {
         navigator.clipboard.writeText(this.aiProcessedPrompt);
+        this.tooltipText = this.dotMessageService.get('Copied');
+        setTimeout(() => {
+            this.tooltipText = null;
+        }, 1000);
```

Comment:
You can do this by adding the `life`attribute where you use the tooltip
https://primeng.org/tooltip#api.tooltip.props.life

Reply:
https://www.primefaces.org/primeng-v15-lts/tooltip take a look if works.

Reply:
After the first click, I need to clean the text to don't show the tooltip again until the next click.